test/browser/element.vdiff.js

```diff
@@ -39,6 +40,40 @@ const elementTag = defineCE(
 	}
 );
 
+const fixedElementTag = defineCE(
+	class extends LitElement {
+		static get styles() {
+			return css`
+				div {
+					background-color: white;
+					border: 1px solid purple;
+					border-radius: 8px;
+					box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+					box-sizing: border-box;
+					position: fixed;
+				}
+			`;
+		}
+		render() {
+			return html`
+				<div class="vdiff-size">
```

Comment:
This represents typical `core` usage. We'll add the appropriate class to `dialog`, dropdown`, etc.